src/components/plots/areaPlot.ts

```diff
@@ -26,13 +26,32 @@ export module Plots {
       this.attr("stroke", defaultColor);
     }
 
+    public y(): Plots.AccessorScaleBinding<number, number>;
+    public y(y: number | Accessor<number>): Area<X>;
```

Comment:
The above I believe should already be declared on `Plots.Line` and thus this should be unnecessary?